src/utils/local_file_mods.py

```diff
@@ -163,6 +163,22 @@ def rd_mkdir(path):
     return None
 
 
+def filename_amender(filename, config):
+    """Append the survey type to filename if syrvery is PNG."""
+
+    survey_year = config["survey"]["survey_year"]
+    survey_type = config["survey"]["survey_type"]
+    run_id = config["filename_items"]["run_id"]
+    tdate = config["filename_items"]["tdate"]
+
+    if survey_type == "PNP":
+        filename = f"{survey_type}_{survey_year}_{filename}_{tdate}_{run_id}.csv"
+    else:
+        filename = f"{survey_year}_{filename}_{tdate}_{run_id}.csv"
+
+    return filename
+
```

Comment:
I like the name of your function. Can you move it from local file mods to src.utils.helpers.py? It's not a func that has a different form depending on the platform.